src/coreclr/dir.common.props

```diff
@@ -41,15 +41,6 @@
     <PackagesBinDir>$(RuntimeBinDir).nuget\</PackagesBinDir>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <!-- Central place to set the versions of all nuget packages produced in the repo -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">6.0.0</PackageVersion>
-
-    <!-- Set the boolean below to true to generate packages with stabilized versions -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

Comment:
These two properties are used for servicing and I will move them into the right location as a port of: https://github.com/dotnet/runtime/commit/2a51bcb2c1fb29b8ec8d8c4a30cb59e851446b00 to master

Reply:
I'm consolidating these properties across the whole repo. Are you saying that I shouldn't do that for coreclr?

Reply:
I won't merge this in before I've figured out the right fix that still allows to consolidate the properties. Will take another look at it tomorrow.

Reply:
I'm saying for coreclr we don't need to consolidate them. These are already on Versions.props but coreclr needs to handle their servicing independently and that's why we need these for coreclr packages here.

Reply:
Curious, why does coreclr need to service independently than the rest of the repo?

Reply:
because coreclr doesn't want their packages to be stable and shipped to NuGet on every servicing release. I think we discussed this here: https://github.com/dotnet/runtime/pull/45876#discussion_r540409002 in case that explanation helps here.

Reply:
I undid my changes for the sake of getting this in but I believe we should follow-up on a pattern that allows us to service all packages the same way.